static/js/types.js

```diff
@@ -24,7 +24,44 @@ var islands = [
     "Weferlingen (Winter)", // CDLC - GM
     "Virtual Reality"       // Vanilla
 ];
-
+var frameworks = {
+    "Modern FA3 (>= 3.5)": {
+        "uiPriority": 0, // order to appear in dropdown, 0 being closet to top
+        "versions": [    // order in dropdowns, so newer at top makes sense. Needs to be exhaustive for search to work
+            //"FA3 3.5.7",
```

Comment:
Note: this is where new FA3 versions will now be added. I've added the soon-to-be FA3 3.5.7 as a commented example